Code-4-Community/speak-for-the-trees-backend-v2
# Sk Add Get Unapproved Images (Filter Site Images)

## Why

Resolves #218 

This PR resolves the ability to get unapproved site images filtered by upload date, site ID, and neighborhood IDs

## This PR 

- Use existing unapproved_images route
- Detailed here: https://docs.c4cneu.com/sftt/sftt-api-spec/#get-unapproved-images-admin-only

## Verification Steps

- Upload images with https://docs.c4cneu.com/sftt/sftt-api-spec/#upload-site-image-admin-and-owner-only
- GET http://localhost:8081/api/v1/protected/sites/unapproved_images, specifying filter parameters
- Reference the sample postman request below. Note that all filter fields are specified as params, similar to filter sites

<img width="954" alt="Screenshot 2024-02-25 at 12 29 00 PM" src="https://github.com/Code-4-Community/speak-for-the-trees-backend-v2/assets/30560773/6ded3a8a-988a-41c6-bee9-4b9124d0f9f8">
